packages/mui-system/src/Stack/StackProps.ts

```diff
@@ -24,6 +24,14 @@ export interface StackBaseProps {
    * Add an element between each child.
    */
   divider?: React.ReactNode;
+  /**
+   * If `true`, the CSS flexbox `gap` is used instead of applying `margin` to children. CSS `gap` removes the [known limitations](https://mui.com/joy-ui/react-stack#limitations),
```

Comment:
```suggestion
   * If `true`, the CSS flexbox `gap` is used instead of applying `margin` to children. While CSS `gap` removes the [known limitations](https://mui.com/joy-ui/react-stack#limitations),
```

Reply:
Updated!